src/aerie_cli/aerie_client.py

```diff
@@ -1488,30 +1488,25 @@ def get_scheduling_goals_by_specification(self, spec_id):
             scheduling_specification_goals(where: {
                 specification_id:{_eq:$spec}
             }){
-                goal{
+                goal_metadata {
                     id
-                    model_id
-                    name
-                    description
-                    author
-                    last_modified_by
-                    created_date
-                    modified_date
                 }
```

Comment:
Does it break anything to exclude all of this information that used to be included?